src/theme/components/tagSet.ts

```diff
@@ -84,7 +84,10 @@ const TagSetExplore = defineStyleConfig({
     py: "xxxs",
     px: "s",
     textDecoration: "underline",
-    transition: "0.3s ease",
+    textDecorationStyle: "dotted",
+    textDecorationThickness: "1px",
+    textUnderlineOffset: "2px",
```

Comment:
These are the same base styles for the `Link` component. Anyway we can extract those styles out and reuse an object for both?

Reply:
Yes, and I think I've used this in other spots as well.  I've been thinking about making it a reusable snippet, but I just haven't gopne through with it...yet.  Let me see what I can do.